corehq/apps/accounting/static/accounting/js/credits-tab.js

```diff
@@ -1,13 +1,16 @@
-var CreditFormModel = function () {
-    var self = this;
-    self.rateType = ko.observable("");
-    self.showProduct = ko.computed(function() {
-        return self.rateType() == 'Product';
-    }, self);
-    self.showFeature = ko.computed(function() {
-        return self.rateType() == 'Feature';
-    }, self);
-};
+var $form = $('#credit-form');
+if ($form.length) {
+    var CreditFormModel = function () {
```

Comment:
feels a bit weird to conditionally define a model. this is the only place it's used?